tasks/pipeline.py

```diff
@@ -243,11 +243,11 @@ def run(
 ):
     """
     Run a pipeline on the given git ref (--git-ref <git ref>), or on the current branch if --here is given.
-    By default, this pipeline will run all builds & tests, including all kitchen tests, but is not a deploy pipeline.
+    By default, this pipeline will run all builds & tests, including all kmt tests, but is not a deploy pipeline.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
    By default, this pipeline will run all builds & tests, including all kmt and e2e tests, but is not a deploy pipeline.
```